### ISSUE TITLE
NOMKSTREAM missing from xadd

### ISSUE THREAD

**Issue by sgarciac** [redis-rs/redis-rs#715]
Hello, currently the NOMKSTREAM option seems to be missing from the implementation of xadd.

**Comment by sgarciac** [comment]
Sure, but I don't know how to tackle the interface. There are currently 4 "xadd" methods (xadd, xadd_maxlen, xadd_map, xadd_maxlen_map). Perhaps adding two more general methods, "xadd_options", "xadd_map_options" would do?

**Comment by djc** [comment]
Yeah, that seems fine to me. It's probably easier for me to judge a PR.

**Comment by somechris** [comment]
It seems the missing functionality described here got implemented in #1361.

Can this be closed or is something still missing?

**Comment by sgarciac** [comment]
we can close
